scripts/uberenv/spack_configs/config.yaml

```diff
@@ -23,8 +23,8 @@ config:
   template_dirs:
     - $spack/templates
 
-  # install directory layout
-  install_path_scheme: "${ARCHITECTURE}-${COMPILERNAME}@${COMPILERVER}/${PACKAGE}@${VERSION}"
```

Comment:
should this be commented out?

Reply:
Yes the default install path scheme is better, but I want to keep here as "documentation".